src/scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/dao/RecurringScheduleDaoImpl.java

```diff
@@ -27,12 +28,16 @@ public List<RecurringScheduleEntity> findAllRecurringSchedulesByAppId(String app
 
   @Override
   @Transactional(readOnly = true)
-  public List getDistinctAppIdAndGuidList() {
+  public List<Pair<String, String>> getDistinctAppIdAndGuidList() {

```

Comment:
is this not just a map ?

Reply:
You are probably right, I  was in a meeting while mindlessly placating warnings. Thanks. :rofl: